tools/pv_tree_generator/build_tree.py

```diff
@@ -1,243 +1,329 @@
-# Copyright 2020 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""build the property-value tree """
-
+from typing import Dict, List, Tuple
+from util import PopObsSpec, StatsVar, read_pop_obs_spec, read_stat_var
 import collections
-import util
 import text_format
+from constants import VERTICALS
+import copy
+
+
+class Root:
```

Comment:
if there is no class variable, then maybe no need to create a class? can just have function get_root_children() and get_root_name()

Reply:
changed to functions.